flow/templates/base_status.jinja

```diff
@@ -27,8 +28,8 @@
 {% endblock%}
 
 {% block operation_summary %}
-| operation | number of eligible jobs |
-| --------- | ----------------------- |
+| operation | number of eligible scheduler jobs |
```

Comment:
This is related to previous discussions where we've considered names like `ExecutionUnit` or `SubmissionTask`, we need to create a separate issue for this because we do need a name for that unit of work.

Reply:
Then I believe that we should not make this change in this PR. I'll revert the related changes.